### PR TITLE
Mhv 46079/filter box case fix

### PR DESCRIPTION
**Pull request by KyleCardwell**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- The filter box contained a list that did not conform to VA standards (the first word in the list should be capitalized)
- Capitalization from the search/category inputs in the filter box was not carried through to the results description, potentially causing confusion to users by mis-representing what was searched for.

### [MHV-46079](https://jira.devops.va.gov/browse/MHV-46079) - SHOULD -Filter message-capitalization To-The content has spelling, punctuation, or typo errors. (05.02.1) #59469 ###

> SM on VA.gov: SHOULD - The content has spelling, punctuation, or typo errors. (05.02.1) #59469
>
> Point of Contact/Reviewers
> Cheryl Evans - @CherylEvans - Content
> 
> *For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.
> 
> —
> 
> Platform Issue
> The content has spelling, punctuation, or typo errors.
> Issue Details
> Filter message box verbiage: "Enter information from one of these fields: to, from, message ID, or subject" "To" should be capitalized as it starts the list.
> Link, screenshot or steps to recreate
> https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51115_Cheryl_1_Capitalization_Screen%20Shot%202023-05-31%20at%2010.44.03%20AM.png
> VA.gov Experience Standard
> [Category Number 05, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)
> Other References
> Platform Recommendation
> Capitalize "to"
> 
> —
> 
> VFS Guidance
> Close the ticket when the issue has been resolved or validated by your Product Owner
> If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
> Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
> If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
> 
> User Story: As a Secure Messaging User, I want to so that .
> 
> Feature Flag Y/N ?
> Manual Testing Y/N Y
> Automated Testing Y/N Y
> Accessibility Testing Y/N ?

## Testing done

- manual testing
- updated unit and cypress tests

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |Screenshot 2024-03-19 at 8 55 19 AM|Screenshot 2024-03-19 at 8 58 50 AM|

## What areas of the site does it impact?

VA.gov - Secure Messaging

## Acceptance criteria

- AC1 Filter box message, "To" should be capitalized
- AC2 results description from filter should be same case as what is searched for (search term and Category)

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by alex-morgun-1 — DISMISSED** [review]
This looks great 👍🏻

File: src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx [hunk 1]
```diff
@@ -166,11 +164,12 @@ const SearchForm = props => {
         in <strong>{folderName}</strong> for{' '}
         {keyword && (
           <>
-            "<strong>{keyword}</strong>"
+            "<strong>{query.queryData.searchTerm}</strong>"
```
Comment (vsaleem) [inline src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx:167]: I saw similar lines that have a null safety. Does this line need one as well?

**Review by vsaleem — DISMISSED** [review]

File: src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx (hunk 1 above)
Comment (KyleCardwell) [reply src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx:167]: great catch!  I'll include that as well

**Review by vsaleem — APPROVED** [review]
Looks good😃

**Review by alex-morgun-1 — APPROVED** [review]

**Pull request merged**